### ISSUE TITLE
runki -Ua (perf , perf annotate ) - no warning when perf not available

### ISSUE THREAD

**Issue by PeterPitterling** [HewlettPackard/LinuxKI#66]
In case perf is not available "**No Warning message**"

```
/opt/linuxki/runki -RUa
=== runki for Linux version 7.8
=== Getting Initial NUMA statistics information ...
=== Installing Linux KI trace points ===
=== Starting: Linux KI LiKI trace dump for 20 seconds ===
===
Command line: /opt/linuxki/kiinfo -likidump dur=20,nop,nop,nop,nop,nop,nop,nop,msr,debug_dir=/sys/kernel/debug,sysignore= -ts 1207_1309

kiinfo (7.8)

kiinfo: spooling trace data to disk...
kiinfo: Tracing complete
=== LinuxKI LiKI dump complete  ===
=== KI trace complete.  Collecting supplemental files.
=== WARNING: NO_TTWU_QUEUE is set in sched_features file ===
=== Running lsof ...
=== Collecting task stack traces from /proc ...
=== Collecting task maps from /proc ...
=== Collecting task numa_maps from /proc ...
=== Collecting task cgroups from /proc ...
=== Collecting binary executable and library symbol table information ...
=== Getting Processor C-state information ...
=== Getting disk information ...

```

In case perf is available "**=== Collecting 20 second perf sample ...**"

```
/opt/linuxki/runki -RUa
=== runki for Linux version 7.8
=== Getting Initial NUMA statistics information ...
=== Installing Linux KI trace points ===
=== Starting: Linux KI LiKI trace dump for 20 seconds ===
===
Command line: /opt/linuxki/kiinfo -likidump dur=20,nop,nop,nop,nop,nop,nop,nop,msr,debug_dir=/sys/kernel/debug,sysignore= -ts 1207_1328

kiinfo (7.8)

kiinfo: spooling trace data to disk...
kiinfo: Tracing complete
=== LinuxKI LiKI dump complete  ===
=== KI trace complete.  Collecting supplemental files.
=== WARNING: NO_TTWU_QUEUE is set in sched_features file ===
=== Running lsof ...
=== Collecting task stack traces from /proc ...
=== Collecting task maps from /proc ...
=== Collecting task numa_maps from /proc ...
=== Collecting task cgroups from /proc ...
=== Collecting binary executable and library symbol table information ...
=== Getting Processor C-state information ...
=== Collecting 20 second perf sample ...
=== Getting disk information ...
```

**Comment by MarkCRay** [comment]
This will be fixed in LinuxKI version 7.9.   Added the following warning if -U option is used and perf is not found in /usr/bin:

=== WARNING: Cannot collect userspace profiling (perf) as /usr/bin/perf is not found ===

**Comment by MarkCRay** [comment]
Fixed in version 7.9, releaesed on 4/19/24
